tests/tokens/test_ERC1155ownable.py

```diff
@@ -0,0 +1,474 @@
+import pytest
+import brownie
```

Comment:
We don't have Brownie installed by default I don't think, can you do this the same way as our other tests do now?

Reply:
I have to research how to, I only know brownie at this point (learning opportunity :D )

Reply:
It's a journey but I'm seeing light at the end of the tunnel. Half way learning eth-tester :-)

Reply:
ETH-tester tests completed and added. pass 100% on my box